mappings/net/minecraft/entity/passive/BeeEntity.mapping

```diff
@@ -1,44 +1,105 @@
+			ARG 2 entity
 	CLASS class_4478 PollinateGoal
+		FIELD field_20378 pollinationTicks I
+		FIELD field_20379 lastPollinationTick I
+		METHOD method_21819 (Lnet/minecraft/class_2248;)Z
+			ARG 0 block
+		METHOD method_21820 completedPollination ()Z
+		METHOD method_21821 getFlower ()Ljava/util/Optional;
+	CLASS class_4479 BeeWanderAroundGoal
+		METHOD method_21822 getRandomLocation ()Lnet/minecraft/class_243;
+	FIELD field_20353 multipleByteTracker Lnet/minecraft/class_2940;
 	FIELD field_20354 anger Lnet/minecraft/class_2940;
 	FIELD field_20355 targetPlayer Ljava/util/UUID;
+	FIELD field_20356 currentPitch F
+	FIELD field_20357 lastPitch F
 	FIELD field_20358 ticksSinceSting I
 	FIELD field_20359 ticksSincePollination I
 	FIELD field_20360 cannotEnterHiveTicks I
 	FIELD field_20361 cropsGrownSincePollination I
 	FIELD field_20362 flowerPos Lnet/minecraft/class_2338;
 	FIELD field_20363 hivePos Lnet/minecraft/class_2338;
 	METHOD method_21763 setBeeAttacker (Lnet/minecraft/class_1297;)Z
+		ARG 1 attacker
+	METHOD method_21764 (Lnet/minecraft/class_4158;)Z
+		ARG 0 type
 	METHOD method_21769 addParticle (Lnet/minecraft/class_1937;DDDDDLnet/minecraft/class_2394;)V
+		ARG 1 world
+		ARG 2 lastX
+		ARG 4 x
+		ARG 6 lastZ
+		ARG 8 z
+		ARG 10 y
+		ARG 12 effect
+	METHOD method_21770 getBlockInRange (Ljava/util/function/Predicate;D)Ljava/util/Optional;
+		ARG 1 isValid
+		ARG 2 distance
+	METHOD method_21775 setTrackerByte (IZ)V
+		ARG 1 bit
+		ARG 2 value
 	METHOD method_21778 getFlowerPos ()Lnet/minecraft/class_2338;
-	METHOD method_21779 isOriginNotFlower ()Z
+	METHOD method_21779 hasFlower ()Z
 	METHOD method_21780 resetPollinationTicks ()V
 	METHOD method_21781 isAngry ()Z
 	METHOD method_21782 getAnger ()I
 	METHOD method_21783 resetCropCounter ()V
 	METHOD method_21784 hasNectar ()Z
 	METHOD method_21785 hasStung ()Z
+	METHOD method_21786 isNearTarget ()Z
+	METHOD method_21787 isPollinating ()Z
+	METHOD method_21788 onHoneyDelivered ()V
 	METHOD method_21789 canEnterHive ()Z
-	METHOD method_21791 isOriginNotHive ()Z
+	METHOD method_21790 updateBodyPitch ()V
+	METHOD method_21791 hasHive ()Z
 	METHOD method_21792 getCropsGrownSincePollination ()I
 	METHOD method_21793 addCropCounter ()V
-	METHOD method_21797 isFlowerPos (Lnet/minecraft/class_2338;)V
+	METHOD method_21794 isHiveValid ()Z
+	METHOD method_21797 setFlowerPos (Lnet/minecraft/class_2338;)V
 		ARG 1 pos
 	METHOD method_21805 setHasNectar (Z)V
+		ARG 1 hasNectar
 	METHOD method_21806 setHasStung (Z)V
+		ARG 1 hasStung
 	METHOD method_21807 setCannotEnterHiveTicks (I)V
 		ARG 1 ticks
+	METHOD method_21808 setNearTarget (Z)V
+		ARG 1 isNearTarget
 	METHOD method_21809 setAnger (I)V
+		ARG 1 ticks
+	METHOD method_21810 setPollinating (Z)V
+		ARG 1 isPollinating
+	METHOD method_21811 getBodyPitch (F)F
+		ARG 1 tickDelta
+	METHOD method_21812 getTrackerByte (I)Z
+		ARG 1 location
+	METHOD method_21813 findPointOfInterest (I)Ljava/util/Optional;
```

Comment:
Is this the villager point of interest?

Reply:
Yes, Bees use the same system to find Hives and Nests.